README.md

```diff
@@ -54,7 +54,7 @@ As a quick demonstration: follow the commands below to start a local simulation
 
    ```sh
    # Use nix-shell if you do not have a version of Nix that supports Nix flakes.
-   nix develop
+   nix develop --extra-experimental-features "flakes nix-command"
```

Comment:
```suggestion
   nix develop
   
   # Alternately, you can explicitly opt-in to flake support
   nix develop --extra-experimental-features "flakes nix-command"
```

Reply:
`nix develop` will work if you have a version of Nix that supports flakes. If you don't, then you can either a) use the flags you specified here, or b) just use `nix-shell` instead of `nix develop`.

I've moved your comment because a typical flakes-enabled user doesn't need to use those flags.